flare/ase/otf_forceonly.py

```diff
@@ -0,0 +1,331 @@
+"""
```

Comment:
Is this module needed? Looks like a lot of duplication of `ase/otf.py`

Reply:
Oops, thanks for checking that. I guess this file is not supposed to be pushed. I'll remove it.

Reply:
fixed via 44763cf